app/gen-server/lib/homedb/HomeDBManager.ts

```diff
@@ -4462,6 +4466,24 @@ export class HomeDBManager {
     const results = await (options.rawQueryBuilder ?
                            getRawAndEntities(options.rawQueryBuilder, queryBuilder) :
                            queryBuilder.getRawAndEntities());
+
+
+    if (options.checkDisabledUser) {
+      // Disabled users shouldn't be able to even log in, but if they
+      // got this far (for example they have an existing websocket
+      // connexion), they shouldn't be able to have any document
+      // access.
+      const hasDisabled = results.raw.some(
+        entry => 'users_disabled_at' in entry && entry.users_disabled_at !== null
+      );
+
+      if (hasDisabled) {
```

Comment:
I think this is equivalent, and a bit easier to read.
```suggestion
      if (results.raw.some(entry => entry.users_disabled_at)) {
```